tests/generic_inline_admin/tests.py

```diff
@@ -74,22 +74,56 @@ def test_basic_edit_POST(self):
         """
         A smoke test to ensure POST on edit_view works.
         """
+        new_mp3_url = "http://example.com/podcast2.mp3"
+        new_png_url = "http://example.com/logo2.png"
         post_data = {
             "name": "This Week in Django",
             # inline data
             "generic_inline_admin-media-content_type-object_id-TOTAL_FORMS": "3",
             "generic_inline_admin-media-content_type-object_id-INITIAL_FORMS": "2",
             "generic_inline_admin-media-content_type-object_id-MAX_NUM_FORMS": "0",
             "generic_inline_admin-media-content_type-object_id-0-id": "%d" % self.mp3_media_pk,
-            "generic_inline_admin-media-content_type-object_id-0-url": "http://example.com/podcast.mp3",
+            "generic_inline_admin-media-content_type-object_id-0-url": new_mp3_url,
             "generic_inline_admin-media-content_type-object_id-1-id": "%d" % self.png_media_pk,
-            "generic_inline_admin-media-content_type-object_id-1-url": "http://example.com/logo.png",
+            "generic_inline_admin-media-content_type-object_id-1-url": new_png_url,
             "generic_inline_admin-media-content_type-object_id-2-id": "",
             "generic_inline_admin-media-content_type-object_id-2-url": "",
         }
         url = reverse('admin:generic_inline_admin_episode_change', args=(self.episode_pk,))
         response = self.client.post(url, post_data)
         self.assertEqual(response.status_code, 302)  # redirect somewhere
+        self.assertEqual(Media.objects.get(pk=self.mp3_media_pk).url, new_mp3_url)
+        self.assertEqual(Media.objects.get(pk=self.png_media_pk).url, new_png_url)
+
+    def test_basic_edit_POST_as_new(self):
+        """
+        A smoke test to ensure POST on edit_view works.
```

Comment:
If this needs a docstring, it should probably say something to differentiate this from the above test.

I know you copied this from the other test's comment, but I'm not sure "smoke test" is a useful description of what this test is for.